lib/CommandLine.pm

```diff
@@ -363,8 +363,8 @@ END
     if( $self->demo_mode )
     {
         $self->_active_range( 1 );
-        $self->_test_time_override( 5 );
-        $self->_warmup_time_override( 0 );
+        $self->_test_time_override( 10 );  # seconds
```

Comment:
They are unrelated.  I made the change so that demo mode would demo a the warmup and test.